cnv/cnv-release-notes/cnv-release-notes.adoc

```diff
@@ -16,52 +16,42 @@ include::modules/technology-preview.adoc[leveloffset=+2]
-
-* The {product-title} dashboard captures high-level information about clusters.
-From the {product-title} web console, access the dashboard by clicking
-*Home -> Dashboards -> Overview*. Note that virtual machines are no longer listed
-in the web console project overview. Virtual machines are now listed within the
-*Cluster Inventory* dashboard card.
-
-=== Other improvements
-
-* After you install {CNVProductName}, MAC pool manager automatically starts.
-If you define a secondary NIC without specifying the MAC address, the MAC pool
-manager allocates a unique MAC address to the NIC.
-+
-[NOTE]
-====
-If you define a secondary NIC with a specific MAC address, it is possible
-that the MAC address might conflict with another NIC in the cluster.
-====
-
+* View high-level information about your virtual machines by
+using the *Virtual Machines* dashboard in the {product-name} web console.
+Access it by navigating to the *Workloads* -> *Virtual Machines* page and
+selecting a virtual machine.
 
 == Resolved issues
 
-* Previously, if you used the web console to create a virtual machine template
-that had the same name as an existing virtual machine, the operation failed.
-This resulted in the message `Name is already used by another virtual machine`.
-This issue is fixed in {CNVProductName} 2.1.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1717802[*BZ#1717802*])
+* When adding a disk to a virtual machine via the *Disks* tab in the web console,
+the added disk always has a `Filesystem` volumeMode, regardless of the volumeMode
+set in the `kubevirt-storage-class-default` ConfigMap.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1753688[*BZ#1753688*])
+
+* When navigating to the *Virtual Machines Console* tab, sometimes no content
+is displayed. As a workaround, use the serial console.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1753606[*BZ#1753606*])
 
-* Previously, if you created a virtual machine with the Pod network connected in
-`bridge` mode and used a `cloud-init` disk, the virtual machine lost its network
-connectivity after being restarted. This issue is fixed in {CNVProductName} 2.1.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1708680[*BZ#1708680*])
+* When you attempt to list all instances of the {CNVProductName} operator from a
+browser, you receive a 404 (page not found) error.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1757526[*BZ#1757526*])
 
+* If a virtual machine uses guaranteed CPUs, it will not be scheduled, because
+the label `cpumanager=true` is not automatically set on nodes. As a
+workaround, remove the `CPUManager` entry from the `kubevirt-config` ConfigMap.
+Then, manually label the nodes with `cpumanager=true` before running virtual
+machines with guaranteed CPUs on your cluster.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1718944[*BZ#1718944*])
 
 == Known issues
 
+* If you have {CNVProductName} 2.1.0 deployed, you must first upgrade to 2.2.0
+before upgrading {product-title}.
```

Comment:
There are but they are all from 2.1 and fixed. Nelly sent out a question to the OS group so I guess we'll see. In my opinion it would confuse the user.